src/services/auth/sign-in/SignInPage.vue

```diff
@@ -127,11 +128,12 @@ export default {
             }),
             showErrorMessage: vm.$route.query.error === 'error' || computed(() => store.state.display.isSignInFailed),
         });
-        const onSignIn = async () => {
+        const onSignIn = async (userId) => {
```

Comment:
These days, I feel good type is good
```suggestion
        const onSignIn = async (userId: string) => {
```